ext/node/polyfills/http.ts

```diff
@@ -1820,6 +1820,8 @@ export function get(...args: any[]) {
   return req;
 }
 
+export const maxHeaderSize = 16_384;
```

Comment:
The stub works for now but note that this value is configurable https://nodejs.org/api/http.html#httpmaxheadersize

Reply:
We don't support these flags in any capacity right now, so I'm gonna leave it as is for the time being.

Reply:
It can be set in the options bag of http.request() too https://nodejs.org/api/http.html#httprequesturl-options-callback